app/views/projects/edit.html.erb

```diff
@@ -2,28 +2,11 @@
   <div class="grid gap-4 xl:grid-cols-2 2xl:grid-cols-3">
     <!--Edit general details form -->
     <div
-      class="
-        p-4
-        bg-white
-        border
-        border-gray-200
-        rounded-lg
-        shadow-sm
-        2xl:col-span-2
-        dark:border-gray-700
-        sm:p-6
-        dark:bg-gray-800
-      "
+      class="p-4 bg-white border border-gray-200 rounded-lg shadow-sm  2xl:col-span-2 dark:border-gray-700 sm:p-6 dark:bg-gray-800"
```

Comment:
Can you undo these changes, it looks like you edited the file outside of vscode and our formatter so it changed the formatting. If I save in VSCode it puts it back to the before state.

Reply:
Ok fixed